src/app/tables/artifacts-table/artifacts-table.component.ts

```diff
@@ -71,15 +71,28 @@ export class ArtifactsTableComponent implements OnChanges {
       // {
       //   734506: {
       //     ID: 'https://koji.domain.local/koji/buildinfo?buildID=734506',
-      //     See Story: 'https://pipeline.domain.local/kojibuild/734506'
+      //     See Story: 'https://pipeline.domain.local/kojibuild/734506',
       //   }
       //   ...
       // }
       this.linkColumnMappings[formattedArtifact[this.uidColumn]] = {
         [this.uidColumn]: externalUrlPipe.transform(artifact),
-        'See Story': this.getNodeStoryUrl(artifact)
+        'See Story': this.getNodeStoryUrl(artifact),
       };
 
+      // If there are associated advisories, we can add links to those as well.
+      if (artifact.advisories) {
+        if (artifact.advisories[0]) {
+          const urls = {};
+          for (let i = 0; i < artifact.advisories.length; i++) {
```

Comment:
Optional: It'd be nice to use the `forEach` method:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach